Tests/TensorFlowTests/ContextTests.swift

```diff
@@ -50,8 +50,24 @@ final class ContextTests: XCTestCase {
     }
   }
 
+  func testAlphaDropout() {
+    Context.local.learningPhase = .inference
+    let dropout = AlphaDropout<Float>(probability: 0.5)
+    let x = Tensor<Float>(repeating: 1.0, shape: [4, 4])
+    XCTAssertEqual(dropout(x), x)
+    withLearningPhase(LearningPhase.inference) {
+      XCTAssertEqual(dropout(x), x)
+      withLearningPhase(LearningPhase.training) {
+        XCTAssertNotEqual(dropout(x), x)
+      }
+      XCTAssertEqual(dropout(x), x)
+    }
+    XCTAssertEqual(dropout(x), x)
+  }
```

Comment:
The test for `AlphaDropout` should be in LayerTests.swift.
Please see [my similar comment](https://github.com/tensorflow/swift-apis/pull/765/files#r403780571).

Reply:
+1 thank you @t-ae for noticing that!